caddytls/ctmonitor.go

```diff
@@ -132,16 +128,16 @@ func getCaddyCerts() ([]string, map[string]struct{}) {
 
 func getCertSpotterCerts(domainName string, config QueryConfig, biggestID *int, retrievedCerts map[string]CertSpotterResponse) (numOfIssuanceObjects int, issuanceID int, retryAfter string, err error) {
 	// issuanceObjects consists of the issuanceObjects returned from CertSpotter.
-	issuanceID = config.Index
 	var (
 		issuanceObjects []CertSpotterResponse
 		certQuery       string
 	)
+	issuanceID = config.Index
 	certQuery = prepQuery(domainName, config)
```

Comment:
This function returns a full URL, not just a query string; the name is a little misleading. Maybe `reqURL` and `prepareRequestURL` instead?